### ISSUE TITLE
Always target latest SDK for building

### ISSUE THREAD

**Issue by muellermartin** [BlueM/cliclick#43]
Currently I'm using macOS 10.12.1 with Xcode 8.1 and if I import the project and initially try to build it I get an error stating `The run destination My Mac is not valid for Running the scheme 'cliclick'.`
This is caused by the project setting `Base SDK` located under PROJECT > cliclick > Build Settings > Architectures set to `macosx10.7` which for me is showing `SDK not found`.

I'm not that familiar with Xcode (actually this is the first time I'm contributing to a project using it  :sweat_smile:) therefore this might be no good suggestion, but couldn't it help to set it to always use the latest SDK?

Therefore I'm suggesting this:

```diff
diff --git a/cliclick.xcodeproj/project.pbxproj b/cliclick.xcodeproj/project.pbx
index 58db10c..b4b3c4b 100644
--- a/cliclick.xcodeproj/project.pbxproj
+++ b/cliclick.xcodeproj/project.pbxproj
@@ -413,7 +413,7 @@
                ONLY_ACTIVE_ARCH = YES;
                OTHER_CFLAGS = "-DDEBUG=1";
                PREBINDING = NO;
-               SDKROOT = macosx10.7;
+               SDKROOT = macosx;
            };
            name = Debug;
        };
@@ -428,7 +428,7 @@
                MACOSX_DEPLOYMENT_TARGET = 10.7;
                OTHER_CFLAGS = "";
                PREBINDING = NO;
-               SDKROOT = macosx10.7;
+               SDKROOT = macosx;
            };
            name = Release;
        };
```

**Comment by BlueM** [comment]
Makes sense. I’ll merge a PR if you’re willing to create one, otherwise I’ll change that myself. Please let me know.

**Comment by muellermartin** [comment]
I'll create a pull request.

**Comment by BlueM** [comment]
Closed via #48
